contracts/hardhat.config.ts

```diff
@@ -46,12 +46,18 @@ export default {
     target: 'ethers-v5',
   },
   networks: {
+    network: {
```

Comment:
Would it make sense to call this `chain-cli-network`? Or something that would explicitly indicate that this value is owned/used by chaincli?

Reply:
if no other places use this `network`, then it makes sense to give a more specific name. Also, this file is broadly shared across functions/teams. would like to get thoughts from @RyanRHall too.